examples/demos/interfaces_query_demo/main.go

```diff
@@ -13,7 +14,10 @@ import (
 const deviceID = models.ID(80603) // crated-before demo router
```

Comment:
Typo in comment (I'm pretty sure that "crated" should be "created"). Also, why not created the `device` as part of the demo.

Reply:
I don't know, Martin. I was not in Codilime when it was written.

Reply:
First sentence done.

Reply:
Second sentence done as well